src/cdk/stepper/step-icon.ts

```diff
@@ -0,0 +1,41 @@
+/**
+ * @license
+ * Copyright Google Inc. All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+
+import {Directive, Input} from '@angular/core';
+import {CdkStep} from './stepper';
+
+@Directive({
+  selector: 'cdkStepIcon'
+})
+export class CdkStepIcon {
```

Comment:
I don't think this needs a cdk version, the icon is a material design specific piece of UI